# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/media/mediasink/AudioSink.h
# Commit: 078521a5c82b
# Full Hash: 078521a5c82b9af962e6da4cc4afda0d2b10eb28
# Author: Paul Adenot <paul@paul.cx>
# Date: 2022-02-08 21:51:08
# Regressor Bug: 1754006
# File Overlap Count: 1
# Description:
#   Bug 1754006 - Increase the processed media queue size and threshold in AudioSink. r=alwu
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D138028
# ==============================================================================

diff -r 51c6ec31fb1c -r 078521a5c82b dom/media/mediasink/AudioSink.h
--- a/dom/media/mediasink/AudioSink.h	Tue Feb 08 15:57:49 2022 +0000
+++ b/dom/media/mediasink/AudioSink.h	Tue Feb 08 16:07:43 2022 +0000
@@ -129,7 +129,7 @@
       AlignedAudioBuffer&& aBuffer, AudioData* aReference);
   // Add data to the processsed queue return the number of frames added.
   uint32_t PushProcessedAudio(AudioData* aData);
-  uint32_t AudioQueuedInRingBufferUs() const;
+  uint32_t AudioQueuedInRingBufferMS() const;
   uint32_t SampleToFrame(uint32_t aSamples) const;
   UniquePtr<AudioConverter> mConverter;
   UniquePtr<SPSCQueue<AudioDataValue>> mProcessedSPSCQueue;
@@ -153,6 +153,7 @@
 
   Atomic<bool> mProcessedQueueFinished;
   MediaQueue<AudioData>& mAudioQueue;
+  const float mProcessedQueueThresholdMS;
 };
 
 }  // namespace mozilla